### DISCUSSION TITLE
MDX Queires Returning Fewer Fields than Expected

### DISCUSSION THREAD

**Discussion by eu2pey4** [SAP/PyRFC#282] [Q&A] [answered]
I am using PyRFC to pull data from SAP BW Cubes using BAPI_MDDATASET_CREATE_OBJECT, BAPI_MDDATASET_SELECT_DATA, and BAPI_MDDATASET_GET_FLAT_DATA. I pass in my MDX query, and the results are close to what I see in MDXTEST, but a few columns are always missing. I have also tried with RSR_MDX_CREATE_OBJECT and RSR_MDX_GET_FS_DATA with the same results.

Is there any reason this might be happening?

**Comment by bsrdjan** [comment, marked as answer]
Answered here: https://github.com/SAP/PyRFC/issues/283
